docusaurus/docs/iOS/uikit/custom-components.md

```diff
@@ -90,17 +92,42 @@ Then, we have to tell the SDK to use our custom subclass instead of the default
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
         ...
-        Components.default.avatarView = RectAvatarView.self
+        Components.default.avatarView = BorderedAvatarView.self
         ...
     }
 }
 ```
 
 | Before  | After |
 | ------------- | ------------- |
-| ![Default Avatars](../assets/default-avatars.png)  | ![Rect Avatars](../assets/rect-avatars.png)  |
+| ![Default Avatars](../assets/default-avatars.png)  | ![Bordered Avatars](../assets/bordered-avatars.png)  |
 
-And that's it 🎉 as you can see all avatars across the UI are now rectangular.
+And that's it 🎉 as you can see all avatars across the UI are now with a border.
+
+#### Change Avatar only in one view
+
+In the previous example we saw that when we customized the avatar view, it changed every UI component that uses an avatar view. All the components in the `Components` config are shared components, but it is also possible to customize a shared component of a specific view only. Let's imagine that we want to apply the previous customization of a bordered avatar view, but only in the quoted reply view:
+
+```swift
+class CustomQuotedChatMessageView: QuotedChatMessageView {
+
+    lazy var borderedAvatarView = BorderedAvatarView()
+
+    override var authorAvatarView: ChatAvatarView {
+        borderedAvatarView
+    }
+}
+```
+Then, set the custom component:
+```swift
+Components.default.quotedMessageView = CustomQuotedChatMessageView.self
+```
+
+As you can see, we override the `authorAvatarView` property of the `QuotedChatMessageView` component and provide our custom bordered avatar view. It is important that the custom view is created lazily, to avoid creating multiple instances of the same view.
```

Comment:
```suggestion
As you can see, we override the `authorAvatarView` property of the `QuotedChatMessageView` component and provide our custom bordered avatar view. It is important that the overridden `authorAvatarView` is backed by a custom view which is created lazily, to avoid creating multiple instances of the same view when `authorAvatarView` is called.
```